### ISSUE TITLE
Cutter crashes immediately or soon after opening TMS320 firmware image

### ISSUE THREAD

**Issue by 0x80O0oOverfl0w** [radareorg/cutter#2261]
**Environment information**
* Operating System: Windows 10 build 19041 and build 17763
* Cutter version: 1.10.3
* Obtained from:
  - [ ] Built from source
  - [X] Downloaded from release
  - [ ] Distribution repository
* File format: Tried both a TI C6745 and TI C55x "Hello World" ELF file and also flash dump of TI AIS image. I compiled the ELF Hello World files using Code Composer 10.

**Describe the bug**
After selecting binary file and selecting TMS320 as architecture, clicking open results in an immediate crash.

**To Reproduce**

1. Go to 'Open File'
2. Click on 'Select'
3. Click on C55x 'hello world' example [c5510_helloworld.zip](https://github.com/radareorg/cutter/files/4819948/c5510_helloworld.zip)
4. Click on 'Open'
5. Click on 'Open'
6. Click on 'Advanced Options'
7. Click on 'Architecture'
8. Scroll down to 'TMS320'
9. Click on 'Ok'
10. See error

**Expected behavior**
Cutter should open and show dissasembly

**Screenshots**
[cutter_error](https://user-images.githubusercontent.com/7553165/85414816-9756cc00-b53a-11ea-9ee8-681332a55a5b.png)

**Additional context**

**Comment by karliss** [comment]
Cutter shouldn't crash either way but file you attached isn't an ELF file even though you talked about ELF  in problem description.

**Comment by 0x80O0oOverfl0w** [comment]
You're correct, the file I attached for the C55x is a COFF. I have the C6745 ELF if it helps? Sorry for the confusion, have been switching between the two today.
